openshift/dataverse-broker.yaml

```diff
@@ -53,33 +96,62 @@ objects:
           - /opt/dataverse-broker/dataverse-broker
           args:
           - --port
-          - "8080"
+          - "8443"
           - -v
           - "4"
           - --logtostderr
+          - --tls-cert-file
+          - "/var/run/dataverse-broker/dataversebroker.crt" #???
+          - --tls-private-key-file
+          - "/var/run/dataverse-broker/dataversebroker.key" #???
+          - --authenticate-k8s-token
           ports:
-          - containerPort: 8080
+          - containerPort: 8443
           readinessProbe:
             tcpSocket:
-              port: 8080
+              port: 8443
             failureThreshold: 1
             initialDelaySeconds: 10
             periodSeconds: 10
             successThreshold: 1
             timeoutSeconds: 2
           livenessProbe:
             tcpSocket:
-              port: 8080
+              port: 8443
             failureThreshold: 3
             initialDelaySeconds: 10
             periodSeconds: 10
             successThreshold: 1
             timeoutSeconds: 2
-          
+          volumeMounts:
+          - mountPath: /var/run/dataverse-broker
+            name: dataverse-broker-ssl #？？？
+            readOnly: true
+        volumes:
+        - name: dataverse-broker-ssl #???
+          secret:
+            defaultMode: 420
+            secretName: dataverse-broker-ssl #???
+            items:
+            - key: tls.crt
+              path: dataversebroker.crt
+            - key: tls.key
+              path: dataversebroker.key
 
 parameters:
 - description: Dataverse service broker
   displayName: dataverse-broker
   name: IMAGE
   required: true
   value: quay.io/dataverse-broker/dataverse-broker:latest
+
+- description: base64 encoded SSL CA Certificate
+  displayName: base64 encoded SSL CA Certificate
+  name: BROKER_CA_CERT
+  required: true
+  value:
+
+- description: Namespace that the starter pack is deployed to
+  displayName: Namespace
+  name: NAMESPACE
+  value: bdc
```

Comment:
This little guy is my contribution. We should also use this parameter anywhere else there's a mention of the namespace